tests_e2e/playwright/conftest.py

```diff
@@ -0,0 +1,44 @@
+import logging
+import os
+
+import dotenv
+import pytest
+from navigator import Navigator
+
+logger = logging.getLogger()
+
+
+@pytest.fixture(scope="session")
+def _navigator_session(browser_name):
+    """Set up a navigator instance, login with username/password, start
+    a server. Teardown when session is complete.
+    Do not use this for individual tests, use `navigator` fixture
+    for tests."""
+    dotenv.load_dotenv()
```

Comment:
Can you explain why we need the whole `dotenv` package? Is this just for debugging locally? If so, can't we manage without an extra dependency? It seems just loading a Python file should suffice?

Reply:
Are you suggesting I do `from dotenv import load_dotenv` ?

Reply:
No, the question is rather "Do we need dotenv at all"? Unless I'm missing something, we only use it to set some environment variables that we later use inside the test. Can't we just set them manually? We do we need an extra dependency for that?

Reply:
I like it for easy-of-use so that I don't have to set env vars to run a test. I can keep a `.env` in my local repo, have it gitignored, and everything just works. Otherwise, I'd have to manually set them in the py file before I ran locally. Which would then cause a git diff. I added it as `dev` requirement, which I didn't think would be that big a deal. If you take serious issue with it, I can remove.